{"name": "test_login_error[data_info1]", "status": "passed", "attachments": [{"name": "stdout", "source": "e87b4224-0a2c-49ec-b9ab-4d53db58383c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_info", "value": "{'username': '12', 'password': '', 'expected': '请输入正确的手机号'}"}], "start": 1596984070460, "stop": 1596984093268, "uuid": "595f5300-b416-4011-9e8d-c6ab3972ac3a", "historyId": "6d0f4d4e5185ad49427c41d429edfdd7", "testCaseId": "5fd5da3b3ca8fdf751feb802572e5b42", "fullName": "tests.test_login.TestLogin#test_login_error", "labels": [{"name": "tag", "value": "error_test"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LIWEI-PC"}, {"name": "thread", "value": "8480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}